### PR TITLE
Denormalize namespace name to job table

### PR DESCRIPTION
**Pull request by henneberger**
The namespace uuid column and constraint can be dropped, and a new constraint on namespace name can be added in the next release

### CODE REVIEW

**Review by julienledem — APPROVED** [review]

File: api/src/main/resources/marquez/db/migration/V20__alter_jobs_to_add_namespace_name.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE jobs ADD namespace_name VARCHAR(255);
```
Comment (wslulciuc) [inline api/src/main/resources/marquez/db/migration/V20__alter_jobs_to_add_namespace_name.sql:1]: With []() now merged, mind updating the migration version to `V22`?

**Review by wslulciuc — CHANGES_REQUESTED** [review]
Looks good 👍 But, we'll need to update the migration version to `V22` to maintain ordering.

File: api/src/main/resources/marquez/db/migration/V20__alter_jobs_to_add_namespace_name.sql (hunk 1 above)
Comment (henneberger) [reply api/src/main/resources/marquez/db/migration/V20__alter_jobs_to_add_namespace_name.sql:1]: pr #935 was branched off of this commit so this pr can be closed

Comment (wslulciuc) [reply api/src/main/resources/marquez/db/migration/V20__alter_jobs_to_add_namespace_name.sql:1]: 👍

**Pull request closed without merging**